src/components/welcome.vue

```diff
@@ -55,31 +55,40 @@
 <style lang="scss" scoped>
   header {
     position: relative;
-    width: 100%;
+    width: 100vw;
+    height: 100vh;
+    min-height: 100vh;
     text-align: center;
     color: #fff;
-    background-image: url(../assets/images/header.jpg);
-    background-position: center;
+    background-image: url('/static/images/capitol.jpg');
+    background-position: top;
     background-size: cover;
-    min-height: 100%;
   }
 
   .header-content {
     width: 100%;
-    text-align: center;
-    background-color: rgba(0,0,0,0.5);
-    box-shadow: 0 0 200px 100px rgba(0,0,0,0.5);
-    padding: 0 50px;
-    position: absolute;
-    top: 50%;
-    transform: translateY(-50%);
+    height: 100%;
+    background-color: rgba(255, 255, 255, 0.6);
```

Comment:
Would you mind changing the alpha value to `.75`? The contrast is pretty low and it's tough for me to read the blue heading in this section. The WAVE assessment also shows a contrast issue that is addressed with an alpha of .75.

There are still tons of other contrast issues with the site, too. I'll create a separate task to address the existing contrast and accessibility issues.